sed/core/dfops.py

```diff
@@ -193,3 +201,166 @@ def forward_fill_partition(df):
             after=0,
         )
     return df
+
+
+def backward_fill_lazy(
```

Comment:
I believe this and forward fill can be combined into one function, as most of the code they share is very similar. Providing a keyword would allow to switch between ffill and bfill.

Reply:
True, but as I needed to rewrite tests too in this case, I chose the faster route.
Can add an issue to change this in the future.